README.md

```diff
@@ -40,10 +65,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author2**
 
-- GitHub: [@](https://github.com/)
-- Twitter: [@](https://twitter.com/)
-- LinkedIn: [](https://www.linkedin.com/in/)
-
+- GitHub: [@githubhandle](https://github.com/Uchennaibute@github)
+- Twitter: [@twitterhandle](https://twitter.com/Uchennaibute@twitter)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/Uchennaibute@linkedin)
```

Comment:
- Kindly remove unused features in your readme file like the one I selected and customize some information to your project information eg. Your readme must contain a  small description of the project. Please also check the arrangement of the element in your read cause the readme is a very important part of the project and it must look clean and professional. it is the first thing that people who visit your website will see to have some idea about your project :+1: